Code/Tools/ProjectManager/Source/ProjectExportWorker.cpp

```diff
@@ -213,7 +225,31 @@ namespace O3DE::ProjectManager
             return AZ::Failure(error);
         }
 
-        return AZ::Success();
+        // Fetch the output directory if the process was successful
+        // All standardized O3DE scripts end with a log line stating the output directory in quotes as the last word
+        // Use regex to collect that directory, and collect the slice of the word surrounded in quotes
+        QString exportOutputLastLine = exportOutput.split('\n', Qt::SkipEmptyParts).takeLast();
+
+        QRegExp quotedDirRegex("'.*'");
+        int pos = quotedDirRegex.indexIn(exportOutputLastLine);
+        if (pos > -1)
+        {
+            QStringList entries = quotedDirRegex.capturedTexts();
+            QString outputDirQuoted = entries.takeLast();
+            QString outputDir = outputDirQuoted.mid(1, outputDirQuoted.size()-2);
+            if (QDir(outputDir).isAbsolute())
+            {
+                m_expectedOutputDir = outputDir;
+                return AZ::Success();
+            }
```

Comment:
outputDir being absolute should not be treated as an error to the process. The error should be reported in the `GetExpectedOutputPath` function above.